lib/chef/data_bag_item.rb

```diff
@@ -107,9 +107,10 @@ def self.object_name(data_bag_name, id)
     end
 
     def to_hash
-      result = self.raw_data
+      result = self.raw_data.dup
+      #result = self.raw_data
       result["chef_type"] = "data_bag_item"
-      result["data_bag"] = self.data_bag
+      result["data_bag"] = self.data_bag.to_s
```

Comment:
Is this ever not a string? Probably fine either way but good to know when it would be something else.